composer.json

```diff
@@ -8,19 +8,24 @@
         "psr-15",
         "middleware",
         "server",
-        "http"
+        "http",
+        "exception",
+        "error",
+        "formatter",
+        "handler"
     ],
     "homepage": "https://github.com/middlewares/error-handler",
     "support": {
         "issues": "https://github.com/middlewares/error-handler/issues"
     },
     "require": {
-        "php": "^7.0",
+        "php": "^7.1",
```

Comment:
Is there any reason to increment the minimum php version supported? All middlewares support 7.0 (like PSR-15 interfaces), and I don't want to change it without a strong reason.

Reply:
[safe](https://packagist.org/packages/thecodingmachine/safe) requires PHP >= 7.1.

PHP 7.0 will be unsupported on 3 Dec 2018, which is nearly here. Not a lot of value in supporting it, IMO.

Reply:
I don't think it's a good idea to include a dependency only [to execute this code](https://github.com/thecodingmachine/safe/blob/78a5636c9dbf03630c15036f1ee2cb78eeb09239/generated/json.php#L40) and that force us to increment the minimum php version required. Besides, there may be users that install this package and they do not even use that formatter.

Reply:
I disagree, but not worth fighting about.